mite/cli/test.py

```diff
@@ -50,17 +59,22 @@ def _setup_msg_processors(receiver, opts):
     receiver.add_listener(recorder.process_message)
     receiver.add_raw_listener(collector.process_raw_message)
 
-    extra_processors = [
-        spec_import(x)()
-        for x in opts["--message-processors"].split(",")
-    ]
+    extra_processors = [spec_import(x)() for x in opts["--message-processors"].split(",")]
     for processor in extra_processors:
         if hasattr(processor, "process_message"):
             receiver.add_listener(processor.process_message)
         elif hasattr(processor, "process_raw_message"):
             receiver.add_raw_listener(processor.process_raw_message)
         else:
-            logging.error(f"Class {processor.__name__} does not have a process(_raw)_message method!")
+            logging.error(
+                f"Class {processor.__name__} does not have a process(_raw)_message method!"
+            )
+
+
+def _get_http_stats_output(receiver):
+    listeners = receiver.filter_listeners(HttpStatsOutput)
+    assert len(listeners) == 1
```

Comment:
Oh, one more that I forgot the first time (sorry) -- this will fail if `HttpStatsOutput` isn;t one fo the specified processors.  I think we should return `None` from this function in that case -- and skip the error validation (i.e. always just `return` from the scenario test command)